kart/working_copy/postgis.py

```diff
@@ -192,6 +193,21 @@ def _drop_spatial_index(self, sess, dataset):
         # PostGIS deletes the spatial index automatically when the table is deleted.
         pass
 
+    def _initialise_sequence(self, sess, dataset):
+        start = dataset.feature_path_encoder.find_start_of_unassigned_range(dataset)
+        if start:
+            quoted_seq_identifier = sess.scalar(
```

Comment:
It is quoted! Perhaps it has to be because it's the only unambiguous way for postgresql to return a fully qualified identifier - for instance, I think an example of a valid but difficult-to-convey SCHEMA_NAME.SEQUENCE_NAME would be `"one.two"."three.four"`